src/axom/spin/tests/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ if ( RAJA_FOUND AND UMPIRE_FOUND )
     #   C++ exception with description "CUDAassert" thrown in the test body.
     set_property(TEST spin_bvh
                  APPEND
-                 PROPERTY ENVIRONMENT  "CUDA_VISIBLE_DEVICES=1")
+                 PROPERTY ENVIRONMENT  "CUDA_VISIBLE_DEVICES=0") # Zero based indexed, so only first GPU is visible
```

Comment:
Boy do I feel silly.  I realized that it's a selector, not a count.  I'd been thinking "OK, we'll have 1 visible device."

Reply:
Thinking in Fortran?!?

Reply:
copy/paste from the internet!